src/constants/auth.ts

```diff
@@ -0,0 +1,6 @@
+const constant = {
+  PHONE_VALIDATION_ERROR: 'Please enter a valid phone number',
```

Comment:
IMO - the error messages should not be in constant file.

User display string is a localization concern and should be abstracted how localization is presented (I think by default it can use `en.js`). I would recommend to read that up a bit and move string there.